alternator/server.cc

```diff
@@ -385,10 +384,10 @@ static tracing::trace_state_ptr maybe_trace_query(service::client_state& client_
 
 future<executor::request_return_type> server::handle_api_request(std::unique_ptr<request> req) {
     _executor._stats.total_operations++;
-    sstring target = req->get_header(TARGET);
-    std::vector<std::string_view> split_target = split(target, '.');
-    //NOTICE(sarna): Target consists of Dynamo API version followed by a dot '.' and operation type (e.g. CreateTable)
-    std::string op = split_target.empty() ? std::string() : std::string(split_target.back());
+    sstring target = req->get_header("X-Amz-Target");
```

Comment:
This was pre-existing code that I didn't change (just dropped the unhelpful name for the string constant), but I think it's needed because get_header  returns an sstring, not a reference, so I need to save the return value somewhere...

Reply:
From include/seastar/http/request.hh:
```c++
    sstring get_header(const sstring& name) const {
        auto res = _headers.find(name);
        if (res == _headers.end()) {
            return "";
        }
        return res->second;
    }
```
Note that the comment here is wrong, and you're right, Seastar probably could have returned an sstring&, not sstring. But it doesn't :-(

Reply:
in general case that's probably unsafe API, so something like get_header_view() could be added, but yes, unrelated to this patch